### PR TITLE
Update docs/codeflow/working-in-codeflow-ide.md

### PR DESCRIPTION
**Pull request by skube**
Submitted with [WebPublisher](https://pr.new)

The instructions (1) mention replacing "github.com" but the link already had done it. Either change the link, or change the steps.

### CODE REVIEW

**Review by exxon47 — APPROVED** [review]
Looks good

File: docs/codeflow/working-in-codeflow-ide.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Follow these steps:
 
 ## Reviewing a PR with Codeflow IDE 
 
-Let's look at [this suspicious PR](https://pr.new/stackblitz/docs/pull/40).
+Let's look at [this suspicious PR](https://github.com/stackblitz/docs/pull/40).
 
 Follow these steps:
 1. Open the link in a new tab. In the GitHub URL of the repository, replace "github.com" with "pr.new".
```
Comment (sylwiavargas) [inline docs/codeflow/working-in-codeflow-ide.md:70]: Do you think we should add a link here as well so the users know what we mean?

**Comment by sylwiavargas** [comment]
Oh that's a really good point - thank you! I've left one question.

File: docs/codeflow/working-in-codeflow-ide.md (hunk 1 above)
Comment (skube) [reply docs/codeflow/working-in-codeflow-ide.md:70]: Sure, I suppose it doesn't hurt to be explicit. (Also, why is it a _suspcicious_ PR?)

**Comment by sylwiavargas** [comment]
Thank you - it looks good 💕  I'm sorry for the wait!
merging right now!

**Review by sylwiavargas — APPROVED** [review]

File: docs/codeflow/working-in-codeflow-ide.md (hunk 1 above)
Comment (sylwiavargas) [reply docs/codeflow/working-in-codeflow-ide.md]: Ah "suspicious" is used to be playful + it's a nonsense PR after all ("fullsnack"). Feel free to make a different suggestion!

**Pull request merged**